src/components/TrendingMoviesList/TrendingMoviesList.test.tsx

```diff
@@ -21,116 +18,63 @@ jest.mock('react-router-dom', () => ({
-        // build
-        const {getByText} = render(
-          <MemoryRouter>
-            <MockedProvider mocks={mocks} addTypename={false} >
-                <TrendingMoviesList />
-            </MockedProvider>
-          </MemoryRouter>
-            );
-
-        // result
-        const title = 'TRENDING MOVIES';
-
-
-        await wait(0);
-
-        // test
-        expect(getByText(title)).toBeInTheDocument;
-
-    })
-
-    it('should render list', async()=>{
-        // build
-        const {getByText} = render(
-          <MemoryRouter>
-            <MockedProvider mocks={mocks} addTypename={false} >
-                <TrendingMoviesList />
-            </MockedProvider>
-          </MemoryRouter>
-        );
-
-        await wait(0);
-
-        // test
-        expect(getByText('title1').textContent).toBe('title1')
-        expect(getByText('overview2').textContent).toBe('overview2')
-
-    })
-
-    it('should be in the document',async ()=>{
-        // build
-        const {getByText} = render(
-          <MemoryRouter>
-            <MockedProvider mocks={mocks} addTypename={false} >
-                <TrendingMoviesList />
-            </MockedProvider>
-          </MemoryRouter>
-        );
-
-        await wait(0);
-
-        // test
-        expect(getByText('title1')).toBeInTheDocument();
-        expect(getByText('overview1')).toBeInTheDocument();
-
-    })
-
-    it('should return 2 items',async ()=>{
-        // build
-        const {queryAllByTestId} = render(
-          <MemoryRouter>
-            <MockedProvider mocks={mocks} addTypename={false} >
-                <TrendingMoviesList />
-            </MockedProvider>
-          </MemoryRouter>
-        );
-
-        await wait(0);
-
-        // build
-        const numberOfMatchingQueries = queryAllByTestId(/trending/i);
-
-        // test
-        expect(numberOfMatchingQueries.length).toBe(2);
-    })
-
-    it('should return items in order',async ()=>{
-        // build
-        const {queryAllByTestId} = render(
-          <MemoryRouter>
-            <MockedProvider mocks={mocks} addTypename={false} >
-                <TrendingMoviesList />
-            </MockedProvider>
-          </MemoryRouter>
-        );
-
-        await wait(0);
-
-        const numberOfMatchingQueries = queryAllByTestId(/trending/i);
-
-        // test
-        expect(numberOfMatchingQueries[0].getAttribute('data-testId')).toBe('trending-1');
-        expect(numberOfMatchingQueries[1].getAttribute('data-testId')).toBe('trending-2');
-    })
+const ComponentUnderTest = <MemoryRouter>
+    <MockedProvider mocks={mocks} addTypename={false} >
+      <TrendingMoviesList />
+    </MockedProvider>
+    </MemoryRouter>;
+
+describe('TrendingMoviesList tests', () => {
+
+  it('should render a valid title', async () => {
+    // build
+    const { getByTestId } = render(ComponentUnderTest);
+
+    await waitFor(() => getByTestId('carousel-title'));
+    // result
```

Comment:
Add this which ensure the component is rendered - but think we can probably improve this as think we should be testing this in the Carousel component - also suppose we should be testing the Error and Loading states